crm/urls.py

```diff
@@ -27,8 +27,9 @@
     path('home/', HomePageView.as_view(), name='home-page'),
     path('client/', include('client.urls', namespace='client')), 
     path('accounts/', include('accounts.urls', namespace='accounts')), 
-    path('api/accounts/', include('accounts.urls', namespace='accounts-api')),
-    path('api/client/', include('client.urls', namespace='client-api')),
+    path('api/v1/', include('api.urls', namespace='api')),
+    path('api/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'), 
```

Comment:
The defaults only allow login with username, update it to allow login with both email and username